ts/features/pn/utils/index.ts

```diff
@@ -0,0 +1,44 @@
+import I18n from "../../../i18n";
+import {
+  NotificationStatus,
+  NotificationStatusEnum
+} from "../store/types/types";
+
+export function getNotificationStatusInfo(status: NotificationStatus): {
+  label: string;
```

Comment:
Do we need to return an object with a string instead of the string directly?

Reply:
@francescopersico actually that’s a stripped down version of a util function from pn-frontend. I’m not sure we won’t need a more detailed description for each state below the label, so I kept it extendable.